runtime/onert/core/include/util/Config.lst

```diff
@@ -31,8 +31,7 @@ CONFIG(USE_SCHEDULER           , bool         , "0")
 CONFIG(TRACING_MODE            , bool         , "0")
 CONFIG(MINMAX_DUMP             , bool         , "0")
 CONFIG(FP16_ENABLE             , bool         , "0")
-CONFIG(RUY_THREADS             , int          , "-1")
-CONFIG(XNNPACK_THREADS         , int          , "-1")
+CONFIG(ONERT_THREADS           , int          , "-1")
```

Comment:
(optional) What about removing `ONERT_` as other variables? We may use `NUM_THREADS`.
```suggestion
CONFIG(NUM_THREADS           , int          , "-1")
```
It seems our python code also uses the name of `num_threads`.